src/channel/sink_impl.rs

```diff
@@ -74,3 +75,73 @@ impl<T> Sink<T> for &UnboundedSender<T> {
         Poll::Ready(Ok(()))
     }
 }
+
+pub trait QuickSinkExt<Item>: Sink<Item> {
```

Comment:
It is a public trait and deserves some documents about the two interfaces.

From the name, I guess that `quick_start_send` is called to start the sink, but it is wired that it also requires the item.

From the code, I see that `quick_send` will call `quick_start_send`. It seems that the names are misleading me.

Reply:
I got the reason. It follows the convention used in tokio Sink.

I think name `start_quick_send` is better than `quick_start_send`.

Reply:
A quick way to document them: Copy document from tokio and refer to send/start_send.